tests/functional/u/unnecessary/unnecessary_comprehension.py

```diff
@@ -44,6 +44,6 @@
 my_set = set()
 
 [elem for elem in my_list]  # [unnecessary-comprehension]
-{k: v for k, v in my_dict.items()} # [unnecessary-comprehension]
+items = {k: v for k, v in my_dict.items()} # [unnecessary-comprehension]
 {k: my_dict[k] for k in my_dict} # [consider-using-dict-items]
```

Comment:
Do you know why the node is not correct on this line ?

Reply:
there's a `Comprehension` node inside each `SetComp`, `DictComp` etc, in the `generators=` attribute. That one seems to have incorrect/incomplete location information.